### DISCUSSION TITLE
这里为什么要用个字典呢

### DISCUSSION THREAD

**Discussion by loveq369** [kingslay/KSPlayer#290] [Q&A]
https://github.com/kingslay/KSPlayer/blob/c4e1607bd20ebe49c85c22da4d3e7713b92724e8/Sources/KSPlayer/MEPlayer/MEPlayerItemTrack.swift#L163

fileprivate var decoderMap = [Int32: DecodeProtocol]()

大神，这里为什么要用个字典呢，一个track不应该只有一个decoder吗？

**Comment by kingslay** [comment]
因为要支持m3u8的多码率切换
